flink-metrics/flink-metrics-dropwizard/src/main/java/org/apache/flink/dropwizard/ScheduledDropwizardReporter.java

```diff
@@ -91,6 +91,10 @@ protected ScheduledDropwizardReporter() {
 		return meters;
 	}
 
+	Map<Gauge<?>, String> getGauges() { return gauges; }
```

Comment:
Can you do a new line for the return and add a annotation like `@VisibleForTesting`?